common-util/Contracts/index.jsx

```diff
@@ -136,22 +136,23 @@ export const getMechMinterContract = () => {
 
 export const getServiceContract = () => {
   const { web3, address, chainId } = getWeb3Details();
-  const { serviceRegistry } = address;
-  const contract = new web3.eth.Contract(
-    isL1Network(chainId)
-      ? SERVICE_REGISTRY_CONTRACT.abi
-      : SERVICE_REGISTRY_L2.abi,
-    serviceRegistry,
-  );
+  let contract;
+  if (isL1Network(chainId)) {
+    const { serviceRegistry } = address;
+    contract = new web3.eth.Contract(SERVICE_REGISTRY_CONTRACT.abi, serviceRegistry);
+  } else {
+    const { serviceRegistryL2 } = address;
+    contract = new web3.eth.Contract(SERVICE_REGISTRY_L2.abi, serviceRegistryL2);
+  }
```

Comment:
@mohandast52 please check if that doesn't break the logic. We needed to rename contract names accordingly to be compatible with the test script.